Dockerfile

```diff
@@ -1,17 +1,27 @@
+ARG COMPOSER_VERSION=1.8
 ARG PHP_VERSION=7.2
+FROM composer:${COMPOSER_VERSION} as composer-build
+FROM php:${PHP_VERSION}-cli-alpine
 
-FROM php:${PHP_VERSION}-cli
+RUN apk add --update --no-cache -t .php-build-deps \
+    autoconf \
+    libmcrypt=2.5.8-r7 \
+    git \
+    make \
+    g++ \
+    gcc \
+    openssl-dev \
+    libzip-dev; \
+    pecl install mongodb-1.5.3
```

Comment:
Do we need this fixed version?

Reply:
Also can't we still use the docker-php-ext-install?

Reply:
mongodb-1.5.3 latest stable version in https://pecl.php.net/package/mongodb

Reply:
In original Dockerfile used https://github.com/jenssegers/laravel-mongodb/blob/master/Dockerfile#L10
`pecl install mongodb && docker-php-ext-enable mongodb`